src/util/loanSearch/queryParamUtils.js

```diff
@@ -10,8 +10,25 @@ const lendToFlssSort = new Map([
 	['popularity', 'personalized'],
 	['loanAmountDesc', 'amountHighToLow'],
 	['loanAmount', 'amountLowToHigh'],
+	['amountLeft', 'amountLeft']
 ]);
 
+/**
+ * Check for excluded query params
+ *
+ * @param {Object} query
+ * @returns {boolean}
+ */
+export function hasExcludedQueryParams(query) {
+	// Handle temporary query param exclusions
+	const excludedParams = ['loanTags', 'partner', 'activity', 'lenderTerm'];
```

Comment:
There are most definitely some others we should add. However, this first pass is against the categories shown in our lend drop down.

I'll test some more in dev against categories showing up in lend-by-category